### PR TITLE
Update data-download.md

### PR DESCRIPTION
**Pull request by TheJoeSchr**
Fix wrong path
Add section about fixing wrong docker permission, if user_data is created by docker, it's permission are set to `root`

Thank you for sending your pull request. But first, have you included
unit tests, and is your code PEP8 conformant? [More details](https://github.com/freqtrade/freqtrade/blob/develop/CONTRIBUTING.md)

## Summary
Explain in one sentence the goal of this PR

Solve the issue: #___

## Quick changelog

- <change log #1>
- <change log #2>

## What's new?
*Explain in details what this PR solve or improve. You can include visuals.*

### CODE REVIEW

File: docs/data-download.md [hunk 1]
```diff
@@ -264,7 +264,19 @@ If you are using Binance for example:
 
 ```bash
 mkdir -p user_data/data/binance
-cp freqtrade/tests/testdata/pairs.json user_data/data/binance
+cp tests/testdata/pairs.json user_data/data/binance
```
Comment (xmatthias) [inline docs/data-download.md:267]: i'm not sure about this in general (not your change per se - but this whole line).

This is intended to only serve as a sample - the pairs in this file may not even exist on binance anymore ... 

I'll think about it ... :laughing:

**Comment by TheJoeSchr** [comment]
Let me know if I can help in any way.

I'm using freqtrade for the first time so I try to let my beginners mind
flow back into the documentation to help others as well


__

Typed on a mobile, please excuse any typing mistakes or short-spokeness.

**Review by xmatthias — APPROVED** [review]
LGTM :+1: 

i've changed the wording slightly (which is often faster to just "do" instead of commenting and waiting) - hope you don't mind.

**Comment by TheJoeSchr** [comment]
Not at all, just wanted to create awareness!


__

Typed on a mobile, please excuse any typing mistakes or short-spokeness.

**Pull request merged**